data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -45,11 +46,13 @@
 import org.apache.iceberg.util.Pair;
 import org.apache.iceberg.util.StructLikeSet;
 import org.apache.iceberg.util.StructProjection;
+
```

Comment:
the formatting is off here and in a bunch of other places. Please run `./gradlew spotlessApply`